### PR TITLE
Fixed seekng and added lockscreen seeking

### PR DESCRIPTION
**Pull request by doctor3w**
Title says it all. No more jumping after the lockscreen seeking.

### CODE REVIEW

File: Podcast/Player.swift [hunk 1]
```diff
@@ -199,12 +199,25 @@ class Player: NSObject {
...
+        if let _ = player.currentItem {
+            let newPosition = CMTimeMakeWithSeconds(position, 1)
+            player.seek(to: newPosition, toleranceBefore: kCMTimeZero, toleranceAfter: kCMTimeZero, completionHandler: { (_) in
+                self.delegate?.updateUIForPlayback()
+                self.updateNowPlayingInfo()
+            })
+        }
+    }
+    
     func configureCommands() {
```
Comment (natashaarmbrust) [inline Podcast/Player.swift]: can you make a spec comment for this function

File: Podcast/Player.swift [hunk 2]
```diff
@@ -199,12 +199,25 @@ class Player: NSObject {
     func setProgress(progress: Double) {
         if let duration = player.currentItem?.duration {
             if !duration.isIndefinite {
-                player.currentItem!.seek(to: CMTime(seconds: duration.seconds * min(max(progress, 0.0), 1.0), preferredTimescale: CMTimeScale(1.0)), completionHandler: nil)
-                delegate?.updateUIForPlayback()
+                player.currentItem!.seek(to: CMTime(seconds: duration.seconds * min(max(progress, 0.0), 1.0), preferredTimescale: CMTimeScale(1.0)), completionHandler: { (_) in
+                    self.isScrubbing = false
+                    self.delegate?.updateUIForPlayback()
+                    self.updateNowPlayingInfo()
```
Comment (natashaarmbrust) [inline Podcast/Player.swift:205]: also can you make a spec comment for updateNowPlayingInfo()

**Review by natashaarmbrust — APPROVED** [review]
LGTM. Just add comments to the top of a couple functions.

**Comment by natashaarmbrust** [comment]
For some reason I am getting errors when I am on your specific branch 
screen shot 2017-11-15 at 12 24 58 am

File: Podcast/Player.swift (hunk 2 above)
Comment (doctor3w) [reply Podcast/Player.swift:205]: Sure thing.

**Comment by doctor3w** [comment]
I'm not sure what that error means. Something is up with MP

**Comment by natashaarmbrust** [comment]
Ok let's make a git issue to explore the MP issue in the future (hopefully Apple will have some fixes by then). Feel free to merge in.

**Pull request merged**